operation_add_cluster_node.go

```diff
@@ -0,0 +1,35 @@
+package ravendb
```

Comment:
Let's start creating proper folder structure here.
Mimic the one we have on the JVM, please